doc/ci.md

```diff
@@ -2,60 +2,66 @@
 
 ## Triggers
 
-The pipeline will be executed for every Pull Request and every push to the repository.
+The pipeline will be executed ONLY for pushes to Pull Requests.
 
-If you want the pipeline to be executed again on a Pull Request, you can add a comment saying `jenkins build please`.
-This will most likely be the case if you suspect that something is wrong with the build server infrastructure.
+If you want the pipeline to be executed, you can add a comment saying `jenkins build please`.
```

Comment:
Better describe when/where this should be done. (In PRs, if there are spurious problems)